legend-engine-pure-code-compiled-core/src/main/resources/core/pure/router/store/routing.pure

```diff
@@ -293,8 +293,55 @@ function  meta::pure::router::store::routing::specializedFunctionExpressionRoute
                                                                                                  )
                                                                                           );
                 ^$processedFunction(routingStrategy = ^$routingStrategy(sets = ^$permSetHavingRootClassImpl(sets = $resolvedRootClassSetImplementations->concatenate($subTypeSetImplementations)->removeDuplicates())->concatenate($permSetsNotHavingRootClassImpl)));,
-              | $processedFunction);                                                  
+              | $processedFunction);
       }
+    ),
+    pair(fe:FunctionExpression[1] | $fe.func->in([meta::pure::mutation::execution::mutationFunctions()]),
```

Comment:
You are wrapping the output of `meta::pure::mutation::execution::mutationFunctions()`, which is multiplicity many  already, in an array `[]`.  While code works, lets remove the brackets to make it cleaner.

Reply:
Done.